css/styles.css

```diff
@@ -386,6 +400,94 @@ header {
   border: 2px solid #6070ff;
 }
 
+#project-wrapper {
+  background-color: #0000004d;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  position: fixed;
+  z-index: 120;
+  justify-content: center;
+  align-items: flex-start;
+  font-size: 16px;
+  overflow-y: scroll;
+  padding: 1em 0;
+  box-sizing: border-box;
+  display: flex;
+  opacity: 0;
+  transition: opacity 0.3s ease-out;
+}
```

Comment:
- [ ] Good job with styling, it looks smart. But I think it would be better if you improve the Popup's background by adding a **blur effect** on the background and using the correct **color** like the one used on the Figma template to make your Popup window sticks to the one of the Figma template. I know you can do it :muscle: :muscle:

- [**OPTIONAL**] I think it would be better if you remove the **scrolling** on the sub-background(body sections content) when you are watching the **Popup Window**, this will prevent users to interact with content in the sub-background or body sections content. This will make your mobile menu correspond to the original design and behaviors professional.

Reply:
I didn't use the color on the template because I cannot use the CSS property `mix-blend-mode` since it makes all the content in my card become see-through. Instead, I just gave the background the blur effect and made its `background-color` transparent, but I had to use a different darker color to match the template color better.

Everything else has been added as requested. 👌